.gitignore

```diff
@@ -0,0 +1,127 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Not a big fan of that (don't have it in my other projects), but no strong feelings.